### PR TITLE
Fixed border color of top bar indicator icons

### LINKED ISSUE #170: Border around reputation notification badge
Should it stay white as it is now, or be darker to blend with the current theme. The border definitely draws attention to it, but it may be overkill.

White mode:
a

Dark mode:
b

* **Browser**: Firefox Nightly 70.0a1 2019-07-16
* **Operating System**: Debian Testing 11 x86_64
* **Link to page with the issue**: N/A
* **Screenshot**: Attached above

### PR DESCRIPTION
**Pull request by AfroThundr3007730**
Turned out to be a bit less straightforward than I'd expected, but I got em all.

Fixes #170

### CODE REVIEW

File: stackoverflow-dark.user.css [hunk 1]
```diff
@@ -560,6 +560,22 @@ regexp("^https?:\/\/((?!area51)\w+\.)*?stackexchange\.com.*") {
     background-color: #242424;
     color: #eee;
   }
+  .top-bar .-secondary .-item .-link .indicator-badge,
+  .top-bar .-secondary .-link._danger-indicator::after {
+    border-color: black;
```
Comment (the-j0k3r) [inline stackoverflow-dark.user.css]: Just go with straight forward colors for consistency pls.

Comment (AfroThundr3007730) [reply stackoverflow-dark.user.css]: You mean using `#000` instead of a keyword?

Comment (the-j0k3r) [reply stackoverflow-dark.user.css]: Yes.

Comment (AfroThundr3007730) [reply stackoverflow-dark.user.css]: Fixed on both counts

**Comment by the-j0k3r** [comment]
Thanks =) will make a release shortly after Im done with some impending fixes

**Pull request merged**